[DOCS] note breaking change from 7.8.1 in migration guide (#59642)
elastic/elasticsearch: docs/reference/migration/migrate_7_8.asciidoc

//tag::notable-breaking-changes[]

[discrete]
[[breaking_781_license_changes]]
=== License Information changes

As of the `7.8.1` release

> .Displays enterprise license as platinum in /_xpack 

Reviewer: ```suggestion
.Displays Enterprise license as Platinum in /_xpack 
```

[%collapsible]
====
*Details*

> The <<get-license,`GET /_license`>> endpoint displays Enterprise licenses as

Author: ```suggestion
The `GET /_license` endpoint displays Enterprise licenses as
```

Reverting adding the reference/link because it fails the build.

> The GET /_license endpoint displays "enterprise" licenses as
> "platinum" by default so that old clients (including beats, kibana and
> logstash) know to interpret this new license type as if it were a
> platinum license.

Reviewer: ```suggestion
The <<get-license,`GET /_license`>> endpoint displays Enterprise licenses as
Platinum by default so that old clients (including Beats, Kibana and
Logstash) know to interpret this new license type as if it were a
Platinum license.
```

> This compatibility layer was not applied to the <<info-api,`GET /_xpack/`>>

Author: ```suggestion
This compatibility layer was not applied to the `GET /_xpack/`
```

> However, this compatibility layer was not applied to the GET /_xpack/
> endpoint which also displays a license type & mode. {es-pull}58217[#58217]

Reviewer: ```suggestion
This compatibility layer was not applied to the <<info-api,`GET /_xpack/`>>
endpoint, which also displays a license type and mode. {es-pull}58217[#58217]
```
